package.json

```
@@ -23,9 +23,11 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-vue-pdf417",
   "dependencies": {
-    "@digitalbazaar/bedrock-web-pdf417": "digitalbazaar/bedrock-web-pdf417"
+    "@digitalbazaar/bedrock-web-pdf417": "digitalbazaar/bedrock-web-pdf417",
+    "dynamsoft-javascript-barcode": "8.8.7"
```

I *think* this also works? If so, it looks more deliberate vs. a typo of omission that someone may erroneously change later.

```suggestion
    "dynamsoft-javascript-barcode": "=8.8.7"
```